src/tree/param.h

```diff
@@ -202,6 +206,11 @@ struct TrainParam : public dmlc::Parameter<TrainParam> {
     DMLC_DECLARE_FIELD(split_evaluator)
         .set_default("elastic_net,monotonic,interaction")
         .describe("The criteria to use for ranking splits");
+    DMLC_DECLARE_FIELD(distributed_dask)
```

Comment:
Adding a parameter is a significant decision. Parameters are part of our API, this means they get documented and are very difficult to modify, change or remove in future. Adding a parameter is a last resort if there is no way to make this implementation function without it.